src/deps_info.json

```diff
@@ -110,6 +110,6 @@
   "_embind_register_enum_value": ["free"],
   "setjmp": ["setThrew", "realloc"],
   "longjmp": ["setThrew", "realloc"],
-  "emscripten_longjmp_jmpbuf": ["setThrew", "realloc"],
-  "saveSetjmp": ["setThrew", "realloc"]
+  "siglongjmp": ["setThrew", "realloc"],
```

Comment:
Is this addition of `siglongjmp` an oversight from before?

Reply:
Yes, `siglongjmp` was missing before, but it wasn't noticeable until now.